connect/client/mixins.py

```diff
@@ -60,7 +60,8 @@ def execute(self, method, path, **kwargs):
 
             if self.response.status_code == 204:
                 return None
-            if self.response.headers['Content-Type'].startswith('application/json'):
+            if 'Content-Type' in self.response.headers and \
```

Comment:
This seams to me too much....this can be solved as simple:
if self.response.headers.get('Content-Type', '').startswith('application/json')